app/controllers/secrets_controller.rb

```diff
@@ -66,7 +66,7 @@ def batch
     authorize_many(variables, :execute)
 
     variables.each do |variable|
-      result[variable.resource_id] = get_secret_from_variable(variable)
+      result[variable.resource_id] = get_secret_from_variable(variable).encode("UTF-8", invalid: :replace) 
```

Comment:
What does it mean `invalid: replace`?
It looks like it changes the password content before it returned to a client, means returns unusable password.
I think that if content cannot be returned as a part of json it should not be returned at all.

Reply:
Any particular reason we're adding explicit `utf-8` encoding as part of the upgrade?

Reply:
Fixed, removed the encoding - unnecessary